setup.py

```diff
@@ -22,7 +22,7 @@
         'affine',
         'fiona>=1.8.4,<2.*',
         'shapely>=1.6.3,<2.*',
-        'rasterio>=1.2.0,< 1.3.0',
+        'rasterio>=1.2.0',
```

Comment:
what is the reason not to set `rasterio>=1.3.0` does the code does not break for rasterio==1.2.X ?

Reply:
[rasterio](https://pypi.org/project/rasterio/1.3.3/#files) 1.3.x available for Python >= 3.8 but telluric supports Python >= 3.6